portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,37 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random image to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomImage() {
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  // Pick a random number.
+  let randNum = Math.floor(Math.random() * 7) + 1;
```

Comment:
It would be good to have a param instead of the direct number, i.e.

let numOfImages = 7;
let randNum = Math.floor(Math.random() * numOfImages) + 1;